### PR TITLE
add foreman_userdata to ci tests

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

File: puppet/modules/slave/templates/test_pull_requests_foreman_userdata.json.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (ekohl) [inline puppet/modules/slave/templates/test_pull_requests_foreman_userdata.json.erb]: Looks like a good candidate to make generic.

**Comment by ekohl** [comment]
Thanks! I don't know what else is required to enable this. Anything on github or the prprocessor?

**Comment by timogoebel** [comment]
@ekohl: Yes. The Github repo needs hooks. I don't know about prprocessor. I think, there is a wiki page on redmine that might have more info.

**Comment by ekohl** [comment]
https://github.com/theforeman/prprocessor/pull/73 is what I suspect is needed. @GregSutcliffe maybe this is a good one to go over tomorrow.

**Comment by GregSutcliffe** [comment]
http://projects.theforeman.org/projects/foreman/wiki/PrProcessor covers most of it, I'll merge that change now. @ekohl we can look at the webhook stuff today, yes.

**Pull request merged**